### PR TITLE
Add Database Connection and "Company" table

### LINKED ISSUE #1: Create a webservice
This repository must contain the frontend and backend.
Later we can think of separating them, but not now.

A simple HTML page being server side rendered is enough for now.

> This service must be written in C# using .NET Framework last version

### LINKED ISSUE #2: Modeling the database
We should create the database using some free tool.
The requirements will be added here.

### LINKED ISSUE #6: Prepare webserver to connect to the MySQL Database
We need to make the `webserver` communicate with the database.
You should do a `SELECT 1` to prove it works and print the result in the index view.

# Rules

- The database changes must be applied with migrations
- You have to use the entity framework

# Documents

https://dev.mysql.com/doc/connector-net/en/connector-net-entityframework-core-example.html
https://learn.microsoft.com/en-us/ef/

### LINKED ISSUE #7: Create royal state CRUD
We should have those fields in the real state table:

|Field|Description|
|-|-|
|name|Real state company name|

### PR DESCRIPTION
**Pull request by MatthewLacerda2**
Communicate project with database
-  Connects with MS-SQL Server
- Select 1 working
- Database changes with migrations, project using Entity Framework

select1

"Company" table added

- Fields: Name, City, State, Country, ID Key
- All fields are Required
- All fields except ID are Strings length 20, minimum 3

### CODE REVIEW

File: .gitignore
Comment (pgbezerra) [inline .gitignore:1]: This file should not be deleted.
It will guarantee you do not commit trash, like binaries and other stuff.

**Review by pgbezerra — CHANGES_REQUESTED** [review]
Please restore the gitignore and remove all files that should be not committed.

The PR has 586 changed files, it is much lower.

No binaries can be committed.

https://vegeta2102.medium.com/why-you-should-not-push-binary-files-to-git-and-what-to-do-if-you-do-29f34692cef0

**Comment by pgbezerra** [comment]
Please, update the PR description.
`Matheus` is meaningful for the project.

File: .gitignore
Comment (MatthewLacerda2) [reply .gitignore:1]: Gitignore added, unwanted files removed

File: LICENSE
Comment (pgbezerra) [inline LICENSE:1]: This file should not be deleted

File: README.md
Comment (pgbezerra) [inline README.md:1]: This file should not be deleted.
Please, update the readme, explaining how to run the project.

**Review by pgbezerra — APPROVED** [review]
The name of the project is webserver. You have changed to razor something.
I'll approve this, but it's not good. We can recreate everything cleaner later.

Also, you run this on Windows, right?
I think this will not boot on Linux. So, doing that, you're on you're own.

**Pull request merged**